provisioning/resources/elasticsearch/config/kibana.yml

```diff
@@ -10,13 +10,13 @@ server.host: "0"
 # Use the `server.rewriteBasePath` setting to tell Kibana if it should remove the basePath
 # from requests it receives, and to prevent a deprecation warning at startup.
 # This setting cannot end in a slash.
-server.basePath: "kibana"
+server.basePath: "/kibana"
```

Comment:
Is the forward slash required?

Reply:
yes, I tested now and it doesn't mount at /kibana otherwise

Reply:
Cool thanks for checking!